README.helm.md

```diff
@@ -43,6 +43,7 @@ Here are all the values that can be set for the chart:
     - `stack` (_String_): Stack.
     - `stagingRequirements`:
       - `diskMB` (_Integer_): Disk in MB for staging.
+      - `imageCacheMB` (_Integer_): Persistent disk in MB for image caching
```

Comment:
```suggestion
      - `imageCacheMB` (_Integer_): Persistent disk in MB for caching staging artifacts across builds.
```

Reply:
I'm also wondering if we should name this something else like `buildCacheMB` or `stagingCacheMB`. I get why you called it `imageCacheMB`, but I feel like that's only because the configuration is set on the kpack `Image` resource and doesn't necessarily describe what it is actually for.

Reply:
`stagingRequirements.stagingCacheMB` feels redundant to me, so I'm inclined toward `buildCacheMB`.

Reply:
Updated... PTanotherL